dojo/settings/settings.dist.py

```diff
@@ -1073,6 +1074,7 @@ def saml2_attrib_map_format(dict):
     'Hadolint Dockerfile check': True,
     'Semgrep JSON Report': True,
     'Generic Findings Import': True,
+    'Solar Appscreener Scan': True,
```

Comment:
This line is not necessary, because CWE is not in the hash_code computation

Reply:
I agree. please remove this change only and we can move forward.